src/lighteval/tasks/default_tasks.py

```diff
@@ -8395,7 +8395,7 @@
     few_shots_split=None,
     few_shots_select=None,
     generation_size=1,
-    metric=[Metrics.loglikelihood_acc_single_token, "mcc_single_token"],
```

Comment:
Why did you remove this one?

Reply:
do we still accept metrics as strings ?

Reply:
Not sure but then you could replace by `Metrics.mcc_single_token` no?

Reply:
yes :)